--- protobuf-2.6.1/autogen.sh	2014-10-21 09:06:06.000000000 +0900
+++ autogen.sh	2017-12-21 23:44:39.675403960 +0900
@@ -19,8 +19,9 @@
 # directory is set up as an SVN external.
 if test ! -e gtest; then
   echo "Google Test not present.  Fetching gtest-1.5.0 from the web..."
-  curl http://googletest.googlecode.com/files/gtest-1.5.0.tar.bz2 | tar jx
-  mv gtest-1.5.0 gtest
+  wget https://github.com/google/googletest/archive/release-1.5.0.tar.gz
+  tar xvf release-1.5.0.tar.gz
+  mv googletest-release-1.5.0 gtest
 fi
 
 set -ex
